src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable you-dont-need-lodash-underscore/find, you-dont-need-lodash-underscore/find-index, you-dont-need-lodash-underscore/get, you-dont-need-lodash-underscore/map, you-dont-need-lodash-underscore/values */
```

Comment:
Just to make sure - This is something temporary, right? We put these comments in files that are going to be refactored? 👀 

Maybe it would make sense to add a comment with link to the GH issue where these lodash functions will be removed?

Reply:
Yes it is temporary, these will be handled in some future refactor PRs.

Reply:
Good idea overall, I'm going to add additional comment to the code